Client/src/Pages/FindSupporter/SupporterPanels/supporterCards.js

```diff
@@ -240,7 +240,7 @@ const SupporterCard = (props) => {
               <Typography>{employer}, {title}</Typography>
               <Typography>{office}</Typography>
               <br/>
-              <Typography>Select Appointment Topic:</Typography>
+              <Typography>Help Needed Topics:</Typography>
```

Comment:
How is this related to this PR?

Reply:
I am not sure what this is actually.

Reply:
Then just revert it back to it’s original value. Looks like you made your branch based on some branch other than BackendLambda causing frontend changes to show up.